app/livechat/server/lib/Helper.js

```diff
@@ -86,6 +86,7 @@ export const createLivechatInquiry = ({ rid, name, guest, message, initialStatus
 	const { _id, username, token, department, status = 'online' } = guest;
 	const { msg } = message;
 	const ts = new Date();
+	const maxQueueWaitTimeMinutes = settings.get('Livechat_max_queue_wait_time');
```

Comment:
I don't think we should do this.
First, the new setting is available on EE only and here the system is reading it whenever an inquiry is created.
There are callbacks(EE) that can/should handle it properly.